docs/mobile-apps/app-storage.md

```diff
@@ -227,6 +227,43 @@ caps.SetCapability("app","storage:c8511dd6-38ec-4f58-b8b9-4ec8c23ad882");
 </TabItem>
 </Tabs>
 
+## Using Dependent Apps for a Test
+
+<small><span class="sauceDBlue">Real Devices Only</span></small>
+
+If your real device testing requires your app under test to have access to other apps and you, therefore, need to install those dependent apps and reference them in your tests, you can do so using the `otherApps` capability.
+
+:::note
+Dependent apps inherit the configuration of the main app under test for settings such as `proxy`, `locale`, and `device orientation`, regardless of what settings may have been applied to the app at the time of upload, because the settings are specific to the device under test.
```

Comment:
Maybe we should add some disclaimer/caveat notice here? To explain a little bit in detail what this means for our customers: 

* If they are using an iOS app as the dependent app, and this app for instance is a third party app, which we cannot properly resign, the test will not work as intended by them. 
* If the dependent app is supposed to run in landscape mode, but the main app is configured for portrait, the dependent app will run in portrait mode.